src/Solr/Document/Xslt.php

```diff
@@ -200,4 +206,29 @@ public static function setYearOrder($order)
     {
         self::$yearOrder = $order;
     }
+
+    /**
+     * Returns true if the enrichment field with the given name should
+     * be included in the Solr index, otherwise returns false.
+     *
+     * Note that comparison of field names is performed case insensitive.
+     *
+     * @param string $fieldName The name of the enrichment field
+     * @return bool
+     */
+    public static function indexEnrichment($fieldName)
+    {
+        $config = Config::get();
+
+        if (isset($config->search->index->enrichment->blacklist)) {
+            $configBlacklist = $config->search->index->enrichment->blacklist;
```

Comment:
Die geladene Blacklist sollten wir cachen und dafür eine getEnrichmentBlacklist() Funktion hinzufügen. Bei einer Vollindexierung in einer großen Instanz wird die Funktion unter Umständen millionenfach aufgerufen.